man/systemd.network.xml

```diff
@@ -907,6 +907,28 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[Neighbor] Section Options</title>
+      <para>A <literal>[Neighbor]</literal> section accepts the
+      following keys. Specify several <literal>[Neighbor]</literal>
+      sections to configure several static neighbors.</para>
```

Comment:
this needs a longer explanation. Should mention the word "ARP" for IPv4, and maybe something about "this adds static entries to the neighbor table which establishes bindings between protocol addresses and link layer addresses for hosts sharing the same link" (based on the description in the ip-neighbor(8) man page)

Reply:
Also, mention that this is IPv4 and IPv6 specific

Reply:
This should be clearer now.